src/protocol/PasLS.Symbols.pas

```diff
@@ -714,15 +713,15 @@ procedure AddField(var Source: String; Value: Integer; Terminate: Boolean = true
 end;
 
 const
-  SYMBOL_ENTRY_NAME = 0;
-  SYMBOL_ENTRY_PATH = 1;
+
+//  SYMBOL_ENTRY_NAME = 0;
+//  SYMBOL_ENTRY_PATH = 1;
```

Comment:
I think you stacked the branch on the other one. Maybe merge in the changes which fixed this?

Reply:
Done.